eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToClusteredVariantProcessor.java

```diff
@@ -71,7 +71,7 @@ public DbsnpClusteredVariantEntity process(SubSnpNoHgvs subSnpNoHgvs) throws Exc
                                                         variantRegion.getChromosome(),
                                                         variantRegion.getStart(),
                                                         variantType,
-                                                        subSnpNoHgvs.isSnpValidated());
+                                                        subSnpNoHgvs.isSnpValidated(), null);
```

Comment:
This should obtain the creation date from the `subSnpNoHgvs` object. The logic should be the opposite of that in [this method](https://github.com/EBIvariation/eva-accession/pull/63/files#diff-41d0f41cbc7c1968137ceba50d080afaR119), because we would want to retrieve the RS creation time first, and use the SS as a fallback only if the RS doesn't have one.